lib/zhr_devs/submissions/aggregates/submission.ex

```diff
@@ -81,4 +101,7 @@ defmodule ZhrDevs.Submissions.Aggregates.Submission do
         last_attempt_at: UtcDateTime.new()
     }
   end
+
+  def apply(%__MODULE__{} = state, %TaskDownloaded{}), do: state
```

Comment:
I wonder if there's an automation that proves that every event can be applied.

Reply:
https://github.com/commanded/commanded/blob/master/lib/commanded/aggregates/aggregate.ex#L419

https://github.com/commanded/commanded/blob/master/lib/commanded/aggregates/aggregate.ex#L517